dbfit-java/mysql/src/main/java/dbfit/environment/MySqlEnvironment.java

```diff
@@ -219,7 +220,7 @@ private static int getSqlType(String dataType) {
             ParamDescriptor rd = parser.parseReturnType(returns);
             allParams.put("", new DbParameterAccessor("",
```

Comment:
Here we're still accessing the c-tor directly `new DbParameterAccessor` instead of `createDbParameterAccessor`. Is that intentional?